build/install_xla_in_source_tree.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import sys
+import shutil
+import argparse
+
+from bazel_tools.tools.python.runfiles import runfiles
+
+parser = argparse.ArgumentParser()
+parser.add_argument("target")
+args = parser.parse_args()
+
+r = runfiles.Create()
+
+
+def _is_windows():
+  return sys.platform.startswith("win32")
+
+
+def _copy_so(src_file, dst_dir):
+  src_filename = os.path.basename(src_file)
+  if _is_windows() and src_filename.endswith(".so"):
+    dst_filename = src_filename[:-3] + ".pyd"
+  else:
+    dst_filename = src_filename
+  dst_file = os.path.join(dst_dir, dst_filename)
+  shutil.copyfile(src_file, dst_file)
+
+
+def _copy_normal(src_file, dst_dir):
+  src_filename = os.path.basename(src_file)
+  dst_file = os.path.join(dst_dir, src_filename)
+  shutil.copyfile(src_file, dst_file)
+
+
+def copy(src_file, dst_dir=os.path.join(args.target, "jaxlib")):
+  # NOTE: this might be useful for debugging on Windows to inspect into the

```

Comment:
I'd just remove this kind of debugging code. It's better not to leave it cluttering things up.